### PR TITLE
Added quicklook-pat 1.0.0

### PR DESCRIPTION
**Pull request by theoriginalgri**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/quicklook-pat.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 '41d6d9ffb042490c93d661b95c392a341ecde42f40de6988ae944c64808aac28'
+
+  url "https://github.com/pixelrowdies/quicklook-pat/releases/download/#{version}/PatternQuickLook.pkg"
+  name 'Adobe Photoshop Patterns Quicklook Plugin'
+  homepage 'https://github.com/pixelrowdies/quicklook-pat'
+  license :gratis
+
+  pkg 'PatternQuickLook.pkg'
+
+  uninstall pkgutil: [
```
Comment (vitorgalvao) [inline Casks/quicklook-pat.rb]: No need for an array, just do a string.

File: Casks/quicklook-pat.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask 'quicklook-pat' do
+  version '1.0.0'
+  sha256 '41d6d9ffb042490c93d661b95c392a341ecde42f40de6988ae944c64808aac28'
+
+  url "https://github.com/pixelrowdies/quicklook-pat/releases/download/#{version}/PatternQuickLook.pkg"
```
Comment (vitorgalvao) [inline Casks/quicklook-pat.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/quicklook-pat.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+cask 'quicklook-pat' do
```
Comment (vitorgalvao) [inline Casks/quicklook-pat.rb]: Why `pat` and not `pattern`?

**Comment by vitorgalvao** [comment]
There was a PR template when you submitted. It needs to be there and be filled.

File: Casks/quicklook-pat.rb (hunk 2 above)
Comment (theoriginalgri) [inline Casks/quicklook-pat.rb]: Fixed.

**Pull request merged**